File: backend/src/main/kotlin/jp/hasshi/backend/model/Seed.kt
package jp.hasshi.backend.model

import jp.hasshi.backend.entity.SeedEntiry

data class Seed(
    val id: Long = 0L,
    val username: String,
    val text: String,
    val favorite: Int,
    val comment: Int,
    val tag: Tag,
) {
    fun toEntity(): SeedEntiry{
        return SeedEntiry(
            username = username,
            text = text,
            favorite = favorite,
            comment = comment,
            tag = tag,
        )
    }
}

enum class Tag {
    help,
    tweet
}

File: ios/HSPCommunity/Models/Seed.swift
import Foundation

struct Seed: Decodable, Identifiable {
    let id: Int
    let username: String
    let text: String
    let favorite: Int
    let comment: Int
    let tag: Tag
}

enum Tag: String, Decodable {
    case help = "help"
    case tweet = "tweet"
}
